library/packages/test/update_messages_view_test.rb

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env rspec
+
+require_relative "test_helper"
+require "packages/update_message"
+require "packages/update_messages_view"
+
+describe Packages::UpdateMessagesView do
+  subject(:view) { Packages::UpdateMessagesView.new(messages) }
+
+  let(:messages) do
+    [
+      Packages::UpdateMessage.new("pkg1", "message 1", "/var/adm/path-1", "/var/adm/path-1"),
+      Packages::UpdateMessage.new("pkg2", "message 2", "/var/adm/path-2", "/var/adm/path-2")
+    ]
+  end
+
+  describe "#richtext" do
+    it "concatenates information of all messages in a richtext string" do
+      expect(view.richtext)
+        .to eq("<h1>Packages messages</h1><h2>pkg1</h2><p><em>This message will be available at /var/adm/path-1</em></p><br>message 1<hr>" \
```

Comment:
NP: I'd be less strict about the message formatting here (to make it shorter and be more flexible if we decide to change the formatting later), we should really focus just on the presence of both messages here. I.e.:

```
.to match(/message 1.*message 2/)
```